HW1_DiceGame/app/src/main/java/edu/fullerton/csu/team06/hw1_dicegame/Gameplay_UI.java

```diff
@@ -0,0 +1,61 @@
+package edu.fullerton.csu.team06.hw1_dicegame;
```

Comment:
I actually made this change and was looking for an easy way to rename the class so that it could change every instance of the name, I found refractor and thought neat.  After I finished I saw this comment and laughed!